functest/deposit_transfer_test.go

```diff
@@ -48,7 +48,15 @@ func TestDepositTransferToken(t *testing.T) {
 	}
 	require.NoError(t, err)
 
-	finalBalance := getBalanceNoErr(t, member, member.ref)
+	var finalBalance *big.Int
```

Comment:
use waitForFunction from utils_test

Reply:
i think its better to use `checkBalanceFewTimes` (like at `send_money_test.go`)